QuoteGeneration/quote_wrapper/qgs_msg_lib/qgs_msg_lib.cpp

```diff
@@ -816,6 +818,242 @@ uint32_t qgs_msg_get_type(const uint8_t *p_serialized_msg, uint32_t size, uint32
         goto ret_point;
     }
     *p_type = p_header->type;
+    ret = QGS_MSG_SUCCESS;
+ret_point:
+    return ret;
+}
+
+qgs_msg_error_t qgs_msg_gen_get_platform_info_req(
```

Comment:
Maybe add a comment that caller needs to free the memory allocated.

Reply:
will add